bin/makeDocs.bash

```diff
@@ -3,118 +3,72 @@
 source colors.bash
 source lib.bash
 
-PWD="$(pwd)"
 
-# Get the path to the main repo directory.
-SCRIPT_PATH=$(readlink -f "$0")                     # Path to this script.
-SCRIPT_DIR=$(dirname "$SCRIPT_PATH")                # Path to directory containing this script.
-REPO_ROOT_DIR=$(builtin cd "$SCRIPT_DIR/.." && pwd) # REPO root directory.
-
-echo -e "${GREEN}Generating FarmData2 documentation.${NO_COLOR}."
+# Setting Env Variables
+PWD="$(pwd)" #Current Working Dir
+SCRIPT_PATH=$(readlink -f "$0") # Script Path
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH") # Script Directory
+REPO_ROOT_DIR=$(builtin cd "$SCRIPT_DIR/.." && pwd) #Repository Root Directory
+
+TARGETS=()
+
+# Parsing all command line arguments
```

Comment:
It would be nice if the default behavior, with no arguments, were to generate all of the documentation.

Reply:
Created an adhoc conditional to do so.